src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -51,10 +51,11 @@
 from databricks.labs.ucx.installer.logs import PartialLogRecord, parse_logs
 from databricks.labs.ucx.installer.mixins import InstallationMixin
 
+from ..mixins.fixtures import get_test_purge_time
```

Comment:
this includes `pytest` into the runtime path of UCX installation, which is going to break customers. copy the code. otherwise adjust the `new_installation` fixture, which is a more proper way of doing this.